Sources/Hummingbird/Server/RequestID.swift

```diff
@@ -15,14 +15,15 @@
 import Atomics
 
 /// Generate Unique ID for each request
-struct RequestID: CustomStringConvertible {
+@_spi(HBXCT)
+public struct RequestID: CustomStringConvertible {
```

Comment:
Can we not make this the `package` access modifier?

Reply:
I guess we could. I'll change that

Reply:
Done